config/jobs/preflight/preflight-postsubmits.yaml

```diff
@@ -0,0 +1,108 @@
+presets:
+- labels:
+    preset-preflight-publish-bot-credentials: "true"
+  env:
+  - name: DOCKER_CONFIG
+    value: /etc/pusher-docker-config
+  volumes:
+  - name: pusher-config
+    secret:
+      secretName: preflight-publish-bot
```

Comment:
I am not sure how we handle these secrets. What can I do to create this secret if I want to push to quay.io?

Reply:
They need to be manually provisioned in our 'trusted' build cluster to do this, and a new robot account for preflight will need to be created.